collect.py

```diff
@@ -7,6 +8,16 @@
 from selenium import webdriver
 from selenium.common.exceptions import NoAlertPresentException, NoSuchWindowException, TimeoutException
 
+if sys.platform.startswith("linux") or sys.platform.startswith("linux2"):  # linux
```

Comment:
Nit: Remove this comment, as the line is explanatory.

sys.platform.startswith('linux2') is useless, since you already have sys.platform.startswith('linux') (a string that starts with linux2 obviously also starts with linux)